CHANGES.md

```diff
@@ -1,3 +1,22 @@
+## edge-19.1.3
+
+* Controller
+  * Added `linkerd install-sp` command to install service profiles into the
+    linkerd control plane
+  * Improved API for ListPods (thanks @alenkacz!)
+  * fast-build has been improved and renamed to build-cli-bin
+* Web UI
+  * Improved resource detail pages now show all resource types
+  * Fixed stats not appearing for routes that have service profiles installed
+* CLI
+  * Improved CLI appearance on Windows systems
+  * Improved `linkerd check` output, fixed some bugs in check
```

Comment:
maybe call out #2141 and #2142 separately? with a shout out to the reporter via #2117?

Reply:
argh, this change got lost in a rebase; I'm going to add it into the release notes